swap/src/monero.rs

```diff
@@ -99,6 +99,31 @@ impl Amount {
         self.0
     }
 
+    pub fn as_monero_decimal(&self) -> Decimal {
+        let piconero_decimal = Decimal::from(self.0);
+        let pic_offcet_decimal = Self::piconero_offset_decimal();
+        piconero_decimal / pic_offcet_decimal
+    }
+
+    fn piconero_offset_decimal() -> Decimal {
+        Decimal::from(PICONERO_OFFSET)
+    }
+
+    pub fn max_bitcoin_from_balance(&self, satoshis: u64) -> Result<bitcoin::Amount> {
```

Comment:
I think a more descriptive name would be:


```suggestion
    pub fn max_bitcoin_for_price(&self, ask_price: u64) -> Result<bitcoin::Amount> {
```